app/subscriber/src/features/my-reports/edit/content/stories/ContentForm.tsx

```diff
@@ -152,6 +183,19 @@ export const ContentForm: React.FC<IContentFormProps> = ({
             onContentChange?.(values);
           }}
         />
+        <Formik initialValues={versions} validationSchema={sentimentFormSchema} onSubmit={() => {}}>
```

Comment:
Don't create a sub-form simply to work around the behavior of the FormikSentiment.  Make the component smarter.